build-support/bin/release.sh

```diff
@@ -94,7 +94,7 @@ function run_pex() {
 function run_packages_script() {
   (
     cd "${ROOT}"
-    ./pants --concurrent run "${ROOT}/build-support/bin/packages.py" -- "$@"
+    ./pants --no-pantsd run "${ROOT}/build-support/bin/packages.py" -- "$@"
```

Comment:
Why this?

Reply:
My thinking was that in at least some of these cases the `--concurrent` flag was actually meant-as "please disable `pantsd`". This one didn't have a comment to explain why it was being used: since this is a bash script, it didn't seem likely that this script was "running under" pants.